test/unit/error.test.ts

```diff
@@ -424,4 +397,147 @@ describe('MongoErrors', () => {
       });
     });
   });
+
+  describe('retryable errors', () => {
+    describe('#isRetryableWriteError', () => {
+      const tests: {
+        description: string;
+        result: boolean;
+        error: MongoError;
+        maxWireVersion: number;
+      }[] = [
+        {
+          description: 'a plain error',
+          result: false,
+          // @ts-expect-error: passing in a plain error to test false case
+          error: new Error('do not retry me!'),
+          maxWireVersion: 2.3
+        },
+        {
+          description: 'a MongoError with no code nor label',
+          result: false,
+          error: new MongoError('do not retry me!'),
+          maxWireVersion: 2.3
+        },
+        {
+          description: 'a MongoWriteConcernError with no code nor label',
+          result: false,
+          error: new MongoWriteConcernError({ message: 'empty wc error' }),
+          maxWireVersion: 2.3
+        },
+        {
+          description: 'a MongoWriteConcernError with a random label',
+          result: false,
+          error: new MongoWriteConcernError(
+            { message: 'random label' },
+            { errorLabels: ['myLabel'] }
+          ),
+          maxWireVersion: 0
+        },
+        {
+          description: 'a MongoWriteConcernError with an ExceededTimeLimit code above server 4.4',
+          result: false,
+          error: new MongoWriteConcernError({}, { code: 262 }),
+          maxWireVersion: 9
+        },
+        {
+          description: 'a MongoWriteConcernError with an ExceededTimeLimit code below server 4.4',
+          result: true,
+          error: new MongoWriteConcernError({}, { code: 262 }),
+          maxWireVersion: 2.3
```

Comment:
"below server 4.4" should test the boundary, so use maxWireVersion: 8, otherwise these would all still pass even if we changed the condition in the errors.ts code to test for, say `maxWireVersion: 6`

this goes for all of the examples above (and varying 2.3 and 0 just makes it more confusing to understand the intent); I am also wondering if there should be cases for both above and below the maxWireVersion: 9 for each case where the maxWireVersion should not be relevant, just to prove that it isn't

Reply:
Fixed, used 8/9